### PR TITLE
Add restart support for docker-compose

### PR DESCRIPTION
**Pull request by waveywaves**
Convert docker-compose restart options to Kubernetes restartPolicy
docker-compose docs:
https://docs.docker.com/compose/compose-file/#restart

kubernetes docs:
https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#restart-policy

fixes https://github.com/UffizziCloud/uffizzi_app/issues/140

### CODE REVIEW

**Comment by gadkins** [comment]
Tagging @moklidia for review of this PR

File: core/app/serializers/uffizzi_core/controller/deploy_containers/container_serializer.rb [hunk 1]
```diff
@@ -67,4 +68,18 @@ def healthcheck
...
+    # throw error if restart policy is not string
+    return nil if object.restart.nil?
+
+    restart_policy = object.restart.to_s
+    # case for value of restart_policy is "no", "always", "on-failure", "unless-stopped"
+    case restart_policy
+    when 'no', 'always', 'on-failure'
+      restart_policy
+    when 'unless-stopped'
+      raise UffizziCore::ComposeFile::ParseError, I18n.t('compose.invalid_restart_policy')
```
Comment (moklidia) [inline core/app/serializers/uffizzi_core/controller/deploy_containers/container_serializer.rb:82]: Let's add a default case

File: core/test/fixtures/files/compose_files/compose_with_restart.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+services:
+  redis:
+    image: redis:latest
+
+  hello-world:
+    image: nginx:latest
+    restart: always
```
Comment (moklidia) [inline core/test/fixtures/files/compose_files/compose_with_restart.yml:7]: Let's add a test to check that the option is parsed correctly

File: db/schema.rb [hunk 3]
```diff
@@ -152,6 +152,7 @@
     t.string "command"
     t.string "service_name"
     t.jsonb "healthcheck"
+    t.jsonb "restart"
```
Comment (the38th) [inline db/schema.rb:155]: why restart is jsonb? It's looks like a string

File: core/app/services/uffizzi_core/compose_file/parsers/services/restart_parser_service.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class UffizziCore::ComposeFile::Parsers::Services::RestartParserService
+  class << self
+    def parse(restart_data)
+      return nil if restart_data.nil?
```
Comment (zipofar) [inline core/app/services/uffizzi_core/compose_file/parsers/services/restart_parser_service.rb:6]: We can do `return if restart_data.nil?` without explicitly `return nil`

File: core/app/services/uffizzi_core/compose_file/parsers/services/restart_parser_service.rb [hunk 5]
```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class UffizziCore::ComposeFile::Parsers::Services::RestartParserService
```
Comment (zipofar) [inline core/app/services/uffizzi_core/compose_file/parsers/services/restart_parser_service.rb:3]: Maybe we can rename `RestartParserService` to `RestartService`. We already have word `Parsers` in class name

**Pull request closed without merging**